GovUkDesignSystemComponents/SubComponents/SummaryListRowAction.cshtml

```diff
@@ -0,0 +1,27 @@
+﻿@model GovUkDesignSystem.GovUkDesignSystemComponents.SummaryListRowActionViewModel
+
+@{
+    string classNames =
```

Comment:
Please inline for consistency.
This can be made shorter so it's not too bad:
`"govuk-summary-list__actions " + Model?.Classes`

Reply:
Done, thanks